cgmanifests/cgmanifest.json

```diff
@@ -553,7 +553,17 @@
             },
             "comments": "dlfcn-win32"
          }
+      },
+      {
+         "component": {
+            "type": "git",
+            "git": {
+               "commitHash": "6812205f18ca4ef54372e87e1a13ce4a859434df",
+               "repositoryUrl": "https://github.com/python-pillow/Pillow.git"
+            },
+            "comments": "python-pillow"
```

Comment:
Would you mind making it more specific? Which part of the code uses python-pillow?